core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -596,6 +599,13 @@ case class MisClassificationsPerCategory
   MisClassifications: Seq[ClassCount]
 )
 
+case class ConfusionMatrixPerThreshold
+(
+  Threshold: Double,
+  @JsonDeserialize(contentAs = classOf[java.lang.Long])
+  ConfusionMatrixCounts: Seq[Long]
```

Comment:
lowerCamelCase on these 2 params

Reply:
Seems like that there was a suggestion in the past to use the upper camel case to follow conventions in that file, so leaving as-is.